hive_agent/chat/chat_manager.py

```diff
@@ -52,7 +54,7 @@ async def generate_response(
             assistant_message = "".join([token async for token in response_stream.async_response_gen()])
         else:
             response = await self.llm.achat(last_message.content, chat_history=chat_history)
-            assistant_message = response.message
+            assistant_message = response.response if hasattr(response, 'response') else str(response)
```

Comment:
is this just for ollama responses i.e. will this stay backwards compatible with existing agents?

Reply:
I don't think message is a valid attribute, I get an error.
I've ran it with all the llm types (openai, ollama, mistral, anthropic) and it seems to be working fine.